woocommerce/woocommerce-gutenberg-products-block
# Move Woo Blocks template directories to latest Gutenberg convention

With WordPress/gutenberg#36647, Gutenberg 12.1.0 has changed the convention for the directory paths from `block-templates` and `block-template-parts` to `templates` and `parts` respectively.

We are also moving our repo to that convention.

Fixes #5343

### Note

Tagging also @tjcafferkey as a reviewer, just to page him (as he had requested this PR to be split in two); however, he's going to be AFK for the next week, so it's really just a CC.

### Testing

#### Manual Testing

How to test the changes in this Pull Request:

1. Open Appearance → Editor → Templates
2. Make sure all the templates and parts appear correctly in line with what's available within our directories
3. Make sure the Mini Cart renders correctly (as it is a template part). Try with and without Gutenberg enabled.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).

### Changelog

> Update the block theme folders to latest Gutenberg convention (i.e. `templates` and `parts`).
